api/CanvasRenderingContext2D.json

```diff
@@ -2445,7 +2445,7 @@
               "version_added": false
             },
             "ie": {
-              "version_added": null
+              "version_added": false
```

Comment:
This is setting `imageSmoothingQuality` to false. I find no indication that `msImageSmoothingQuality` was ever a thing, so this seems right.